File: pgm/ABC_C#/ABC_143_A.cs
using System;
// using System.Collections.Generic;
// using System.Linq;
// using System.Text;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        int A, B;
        A = int.Parse(arr[0]);
        B = int.Parse(arr[1]);
        Console.WriteLine(2 * B < A ? A - 2 * B : 0);
        return;
    }
}

File: pgm/ABC_Visual_Basic/ABC_143_A.vb
' Imports System.Collections.Generic
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim A, B As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        A = arr(0) : B = arr(1)
        Console.WriteLine(If(2 * B < A, A - 2 * B, 0))
        Return
    End Sub
End Class